bc_obps/registration/tests/endpoints/v2/_operations/_operation_id/_registration/test_operation_representative.py

```diff
@@ -171,3 +171,73 @@ def test_register_operation_operation_representative_endpoint_bad_data(self):
 
         # Assert
         assert response.status_code == 422
+
+
+class TestOperationRepresentativePutEndpoint(CommonTestSetup):
+    def test_operation_representative_endpoint_unauthorized_roles_cannot_put(self):
+        operation = baker.make_recipe(
+            'utils.operation',
+        )
+        # cas users and unapproved industry users can't put
+        roles = ["cas_pending", "cas_analyst", "cas_admin", "industry_user"]
+        for role in roles:
+            response = TestUtils.mock_put_with_auth_role(
+                self,
+                role,
+                self.content_type,
+                {'id': 1},
+                custom_reverse_lazy("remove_operation_representative", kwargs={'operation_id': operation.id}),
+            )
+            assert response.status_code == 401
+            assert response.json().get('detail') == 'Unauthorized'
+
+    def test_users_cannot_update_other_users_operations(self):
+        # authorize current user
+        baker.make_recipe('utils.approved_user_operator', user=self.user)
+        # create operation not belonging to current user
+        operation = baker.make_recipe(
+            'utils.operation',
+        )
+
+        response = TestUtils.mock_put_with_auth_role(
+            self,
+            "industry_user",
+            self.content_type,
+            {'id': 1},
+            custom_reverse_lazy("remove_operation_representative", kwargs={'operation_id': operation.id}),
+        )
+        assert response.status_code == 401
+        assert response.json().get('message') == 'Unauthorized.'
+
+    def test_remove_operation_representative_endpoint_success(self):
+        approved_user_operator = baker.make_recipe('utils.approved_user_operator', user=self.user)
+        users_operator = approved_user_operator.operator
+        operation = baker.make_recipe('utils.operation', operator=users_operator)
+        contact = baker.make_recipe('utils.contact')
+        users_operator.contacts.add(contact)
+        data = {
+            'id': contact.id,
+        }
+        response = TestUtils.mock_put_with_auth_role(
+            self,
+            "industry_user",
+            self.content_type,
+            data,
+            custom_reverse_lazy("remove_operation_representative", kwargs={'operation_id': operation.id}),
+        )
+        assert response.status_code == 200
```

Comment:
The response.json() is {id: None}. Not sure why--as far as I can tell I'm not returning anything